File: Student-Tracker-App/src/models/enrolment.ts
export interface Enrolment {
    id: string; // Guid → string
    institution?: string;
    qualification?: string;
    qualificationType?: string;
    enrolmentDate?: string; // DateTime → ISO date string (or use Date if parsed)
    averageToDate?: number; // int → number
    studentId: string; // Guid → string
  }

File: StudentTracker/StudentTracker/Entities/Enrolment.cs
namespace StudentTracker.Entities
{
    public class Enrolment
    {
        public Guid Id { get; set; }
        public string? Institution { get; set; }
        public string? Qualification { get; set; }
        public string? QualificationType { get; set; }
        public DateTime EnrolmentDate { get; set; }
        public int AverageToDate { get; set; }
        public Guid StudentId { get; set; }
    }
}
